examples/basic/bootstrap.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+from pathlib import Path
+import time
+
+from oremda.clients import Client
+from oremda.clients.utils import run_container
+from oremda.registry import Registry
+
+
+SINGULARITY_DIR = Path.cwd() / '.singularity'
+
+
+
+def main():
+    client = Client('docker')
+
+    if client.type == 'docker':
+        plasma_image = f'oremda/plasma'
```

Comment:
Just to document here what we mentioned in our discussion: this bootstrap code may be able to be combined with the runner code, and the plasma image code may be put in the runner code as well. This could simplify things.

Reply:
Yes, I'll try to combine them together, maybe in this PR already